tests/phpunit/Unit/mw.smw.info.tests.lua

```diff
@@ -0,0 +1,44 @@
+--[[
+	Tests for smw.info module
+
+	@since 1.0
+
+	@licence GNU GPL v2+
+	@author oetterer
+]]
+
+local testframework = require 'Module:TestFramework'
+
+-- Tests
+local tests = {
+	{ name = 'info (no argument)', func = mw.smw.info,
+		args = { '' },
+		expect = { nil }
+	},
+	{ name = 'info (text, no icon)',
+		func = function ( args )
+			local ret = mw.smw.info( args )
+			return type(ret), #ret > 0
+		end,
+		args = { 'Infotext' },
+		expect = { 'string', true }
```

Comment:
So all assertions here basically are just checking that the result is a non-empty string? (I'm not sure since I'm not familiar with this rather odd way of writing lua tests (which I'm guessing is somewhat forced upon you by MediaWiki).)